website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -156,8 +156,8 @@ Some of these buckets have cross-account IAM policies attached so that they can
 Hitting 'play' next to `Deploy DR ORCA buckets`, `Deploy Dev RDS Stack` and `Deploy Dev Cumulus and ORCA Stack` brings up a checkbox list to run multiple jobs at once. Note that none of the checkboxes should be checked.
 :::
 
-The Cumulus and TF buckets as well as dynamoDB table in cumulus OU account are created automatically in the `Deploy RDS cluster` and `Deploy Dev Cumulus and ORCA Stack` stages.
-These are the buckets that will be created in cumulus OU account:
+The Cumulus and TF buckets as well as dynamoDB table in cumulus OU account are created automatically in bamboo `Deploy ORCA Buckets and Cumulus/ORCA modules` stage. 
+These are the buckets that need to be created in cumulus OU account:

```

Comment:
This was an intentional change on my part. "buckets need to be created" implies that the user must handle creation. "buckets will be created" states that they will be created without additional instruction.

Reply:
I concur

Reply:
fixed